### PR TITLE
doc: create readme

### PR DESCRIPTION
**Pull request by samen93**

### CODE REVIEW

**Comment by samen93** [comment]
@mariux  i didn't add var "region" as it has no usage in the main.tf

File: README.md [hunk 1]
```diff
@@ -1 +1,291 @@
...
+
+  The location for the KeyRing. A full list of valid locations can be found by running `gcloud kms locations list` or in the docs: https://cloud.google.com/kms/docs/locations.
+
+- **`keyring`**: **_(Required `string`)_**
+
+  The resource name for the KeyRing.
+
+- **`keys`**: _(Optional `list(keys)`)_
+
+  A list keys to be created. Each name belonging to the specified Google Cloud Platform KeyRing and match the regular expression `[a-zA-Z0-9_-]{1,63}`.
```
Comment (katcipis) [inline README.md:99]:
```suggestion
  A list of key objects describing how keys are going to be created.`. 
```

And maybe move the key.name docs to the **name** attribute documentation.

File: README.md [hunk 2]
```diff
@@ -1 +1,291 @@
...
+  A list keys to be created. Each name belonging to the specified Google Cloud Platform KeyRing and match the regular expression `[a-zA-Z0-9_-]{1,63}`. 
+  
+  Default is `[]`.
+
+  Example
+
+  ```hcl
+  keys = [{
+    name       = "terraform-state-bucket-key"
+    owners     = ["terraforn@example-project.iam.gserviceaccount.com"]
```
Comment (katcipis) [inline README.md:108]:
```suggestion
    owners     = ["terraform@example-project.iam.gserviceaccount.com"]
```

terraforn is not necessarily wrong, but it does seem to be a typo =P

File: README.md [hunk 3]
```diff
@@ -1 +1,291 @@
...
+    encrypters = ["group:one@example.com","group:two@example.com"]
+    decrypters = ["group:three@example.com"]
+  }]
+  ```
+
+  Each `keys` object accepts the following fields:
+
+  - **`name`**: **_(Required `string`)_**
+
+    The name of a specified Google Cloud Platform KeyRing.
```
Comment (katcipis) [inline README.md:118]:
```suggestion
    The name of a specified Google Cloud Platform KeyRing. It must belong to the specified Google Cloud Platform KeyRing and matching the regular expression `[a-zA-Z0-9_-]{1,63}
```

**Review by katcipis — COMMENTED** [review]
Some possible small improvements

**Review by mariux — APPROVED** [review]

**Pull request merged**